content/doc/book/system-administration/reverse-proxy-configuration-with-jenkins/reverse-proxy-configuration-lighttpd.adoc

```diff
@@ -0,0 +1,103 @@
+---
+layout: subsection
+---
+
+ifdef::backend-html5[]
+ifndef::env-github[:imagesdir: ../../../resources/managing]
+:notitle:
+:description:
+:author:
+:email: jenkinsci-users@googlegroups.com
+:sectanchors:
+:toc: left
+endif::[]
+
+[[running-jenkins-behind-lighttpd]]
+= Reverse proxy - Lighttpd
+
+In situations where you have existing web sites on your server, you may
+find it useful to run Jenkins (or the servlet container that Jenkins
+runs in) behind https://www.lighttpd.net/[Lighttpd], so that you can bind Jenkins
+to the part of a bigger website that you may have or just centralize the TLS
+termination in a single place.
```

Comment:
```suggestion
In situations where you have existing web sites on your server, you may find it useful to run Jenkins (or the servlet container that Jenkins runs in) behind https://www.lighttpd.net/[Lighttpd].
This allows you to bind Jenkins to the part of a bigger website that you may have or centralize the TLS termination in a single place.
```
Ideally one sentence per line (which makes it easier to review), but thought this might be a bit long so separated it into two sentences. I believe the message remains the same, but please let me know if there's any sort of discrepency!